fog/ledger/enclave/api/Cargo.toml

```diff
@@ -25,4 +25,4 @@ fog-types = { path = "../../../fog_types" }
 
 # third-party
 displaydoc = { version = "0.2", default-features = false }
-serde = { version = "1.0", default-features = false, features = ["alloc", "derive"] }
+serde = { version = "1.0", default-features = false, features = ["alloc", "derive"] }
```

Comment:
@tubi70 is it possible that you can change you text editor settings to not remove line breaks at end of file? this is creating a lot of noise in the diffs and will make PRs harder to review